testsuite-shading/pom.xml

```diff
@@ -49,8 +49,8 @@
   <build>
     <extensions>
       <extension>
-        <groupId>kr.motd.maven</groupId>
-        <artifactId>os-maven-plugin</artifactId>
+        <groupId>com.tisonkun.os</groupId>
+        <artifactId>os-detector-maven-plugin</artifactId>
```

Comment:
/cc @trustin

Reply:
Related PR https://github.com/trustin/os-maven-plugin/pull/74

Reply:
I don't have time to maintain it unfortunately 😢